File: Domain/AppUserAccess.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain
{
    public class AppUserAccess
    {
		[Key]
      	[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
	  	public int Id { get; set; }
		public string Title { get; set; }
		public string ActionScript { get; set; }
		public string Description { get; set; }
    }
}

File: client-app/src/features/AppUserAccess/AppUserAccess.ts
export interface IAppUserAccess {
	Id: number
	Title: string
	ActionScript: string
	Description: string
}

export class AppUserAccess implements IAppUserAccess {
	Id: number = 0;
	Title: string = '';
	ActionScript: string = '';
	Description: string = '';
  
  constructor(init?: IAppUserAccess) {
    Object.assign(this, init);
  }
}

